pkg/registry/cachesize/cachesize.go

```diff
@@ -105,6 +105,10 @@ func SetWatchCacheSizes(cacheSizes []string) {
 			glog.Errorf("invalid size of watch cache capabilities: %s", c)
 			continue
 		}
+		if size < 0 {
+			glog.Errorf("invalid size %d of watch cache capabilities: %s", size, c)
```

Comment:
The watch cache size here eventually will be passed into watchCache.capacity, see https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/watch_cache.go#L146

I assume the negative value is not valid to be a slice size?